holobot/extensions/moderation/commands/disable_auto_mute_command.py

```diff
@@ -17,9 +17,9 @@ def __init__(self, warn_manager: IWarnManager) -> None:
         self.required_permissions = Permission.ADMINISTRATOR
         self.__warn_manager: IWarnManager = warn_manager
     
-    async def execute(self, context: SlashContext) -> CommandResponse:
-        await self.__warn_manager.disable_auto_mute(str(context.guild_id))
-        await reply(context, "Users won't be muted automatically anymore.")
+    async def execute(self, context: ServerChatInteractionContext) -> CommandResponse:
+        await self.__warn_manager.disable_auto_mute(context.server_id)
         return AutoMuteToggledResponse(
-            author_id=str(context.author_id)
+            author_id=str(context.author_id),
```

Comment:
Remove the unnecessary string conversions. These are now strings by default. Check other files, too.